components/dashboard/PortfolioSummary.tsx

```diff
@@ -42,7 +44,16 @@ const ChartEntry: FunctionComponent<ChartItem> = ({
 };
 
 const PortfolioSummary: FunctionComponent<PortfolioSummaryProps> = ({ title, data, totalBalance, isProtocol, minSlicePercentage = 0.05 }) => {
-
+const [loading, setLoading] = useState(true); // Add loading state
+  // Simulate data fetching
+  useEffect(() => {
+    const fetchData = async () => {
+      // Simulate an API call
+      await new Promise(resolve => setTimeout(resolve, 2000)); // Simulate delay
+      setLoading(false);
+    };
+    fetchData();
+  }, []);

```

Comment:
What is this? Why are you simulating a loading? Are you to understand the issue ?